auxil/spicy/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Spicy needs the full prefix flex and bison while Zeek captures only the paths
+# to the executables. Derive the prefixes from the binary paths if given under
+# the assumption that their bindir is under their prefix (which also implies
+# that one such prefix even exists).
+
+if ( NOT FLEX_EXECUTABLE )
+   find_package(FLEX REQUIRED)
+endif ()
+
+get_filename_component(dir ${FLEX_EXECUTABLE} DIRECTORY ABSOLUTE)
+set(FLEX_ROOT ${dir}/..)
+
+if ( NOT BISON_EXECUTABLE )
+   find_package(BISON REQUIRED)
+endif ()
+
+get_filename_component(dir ${BISON_EXECUTABLE} DIRECTORY ABSOLUTE)
+set(BISON_ROOT ${dir}/..)
+
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-missing-braces")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-vla")
```

Comment:
This is portable across compilers?

Reply:
This mirrors how the flag is enabled, https://github.com/zeek/cmake/blob/b1c9e55d1e837d46fc36312b567245013cf9646a/SetDefaultCompileFlags.cmake#L10:

```.cmake
set(EXTRA_COMPILE_FLAGS_CXX "-Wno-register -Werror=vla")
```